pipelines/build/common/config_regeneration.groovy

```diff
@@ -396,11 +416,11 @@ class Regeneration implements Serializable {
                     def getPipelines = queryAPI("${jenkinsBuildRoot}/api/json?tree=jobs[name]&pretty=true&depth1")
 
                     // Parse api response to only extract the relevant pipeline
-                    getPipelines.jobs.name.each{ pipeline ->
+                    getPipelines.jobs.name.each { pipeline ->
                         if (pipeline.contains("pipeline") && pipeline.contains(versionNumbers[0])) {
-                            // TODO: Paramaterise this
+                            // TODO: Parametrise this
```

Comment:
a dominant spelling in the UK, uncommon elsewhere...  but all [4 acceptable spellings](https://en.wiktionary.org/wiki/parametrise#English) agree, the 3rd 'a' should never have been there...  ;)